ergo-lib/src/wallet/multi_sig.rs

```diff
@@ -140,6 +140,9 @@ pub fn bag_for_multi_sig(
     simulated_propositions: &[SigmaBoolean],
     proof: &[u8],
 ) -> Result<HintsBag, SigParsingError> {
+    if let SigmaBoolean::TrivialProp(_) = sigma_tree {
```

Comment:
@kushti  If a tree (reduced script) passed in `bag_for_multi_sig` is a boolean (`TrivialProp`), is it ok to return an empty `HintsBag`? It seems like a reasonable thing to do.
Our `bag_for_multi_sig` is implemented against the reference Scala implementation.

Reply:
@kushti Nevermind. I cannot think of any other result in this case besides an empty bag.